configextractor_/configextractor_.py

```diff
@@ -29,6 +30,13 @@
 CONNECTION_USAGE = [k.name for k in ConnUsageEnum]
 
 
+class Base64Encoder(json.JSONEncoder):
+    def default(self, o):
+        if isinstance(o, bytes):
+            return b64encode(o).decode()
```

Comment:
Do you want a limit on the length of the returned string? Maybe truncate it to 1000 characters and add "..." if it's longer?